src/hearings/components/hearing-judge-name/hearing-judge-name.component.html

```diff
@@ -1,4 +1,5 @@
 <xuilib-find-person
+  #personControl
   title=""
```

Comment:
Why have declared an input and nothing is assigned remove the parameter input

Reply:
Removed unused inputs